opencti-platform/opencti-front/src/private/components/common/lists/FiltersElement.js

```diff
@@ -68,12 +81,44 @@ const FiltersElement = ({
               availableRelationshipTypes={availableRelationshipTypes}
               allEntityTypes={allEntityTypes}
               openOnFocus={true}
+              autoSelect={false}
```

Comment:
I think this is a wrong rebase. All this properties are already set in the FilterAutocomplete component.

Reply:
fix